cms/utils/compat/forms.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
```

Comment:
If you talking about `# -*- coding: utf-8 -*-` it is there already. It's not introduced with my PR.

With git blame you can see that the commit introducing it is this https://github.com/divio/django-cms/commit/3ee64f9f470101ac7b336feb19ccef5f31a10c96